pandas/core/reshape/merge.py

```diff
@@ -198,15 +198,15 @@ def merge(
           to SQL left anti join; preserve key order.
         * right_anti: use only keys from right frame that are not in left frame, similar
           to SQL right anti join; preserve key order.
-    on : label or list
+    on : Hashable or a sequence of the previous
         Column or index level names to join on. These must be found in both
         DataFrames. If `on` is None and not merging on indexes then this defaults
         to the intersection of the columns in both DataFrames.
-    left_on : label or list, or array-like
+    left_on : Hashable or a sequence of the previous, or array-like
```

Comment:
Not sure if a sequence of the previous already means `array-like`. No big deal, but I guess it could be removed.